app/js/components/manageApps/ManageApps.jsx

```diff
@@ -125,6 +130,49 @@ export default class ManageApps extends React.Component {
     );
   }
 
+  fetchCurrentUser(){
+    const apiHelper = new ApiHelper(null);
+    const getUserData = new Promise(function(resolve, reject) {
+      apiHelper.get('/v1/session').then(response => {
```

Comment:
How is an `error` condition from the HTTP call handled, since a promise is created locally? Basically, am talking about an `error` resolve.

Reply:
working on it

Reply:
done